0.Preprocess_Data/3.data_aggregation.ipynb

```diff
@@ -10,7 +10,7 @@
         },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        df_nomic = df_nomic.drop(
</pre><p>Why do you drop these columns and the first 25 columns?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>metadata that does not contain metadata in the title</p>